.github/workflows/ci.yml

```diff
@@ -79,7 +79,7 @@ jobs:
         env:
           RAILS_ENV: test
           # REDIS_URL: redis://localhost:6379/0
-        run: bin/rails db:test:prepare test test:system
+        run: bin/rails db:test:prepare spec
```

Comment:
Just fyi, I meant introduce those improvements to the previously opened PR, not merge and create a new one. 

In production app, you only merge once you have explicit approval